streaming/java/streaming-api/src/main/java/org/ray/streaming/jobgraph/JobGraphBuilder.java

```diff
@@ -27,7 +27,7 @@ public JobGraphBuilder(List<StreamSink> streamSinkList, String jobName) {
   }
 
   public JobGraphBuilder(List<StreamSink> streamSinkList, String jobName,
-      Map<String, String> jobConfig) {
+                         Map<String, String> jobConfig) {
```

Comment:
Code indentation

Reply:
I think it's more readable to align arguments as follows:
```java
  public void f(int a1,
                int a2) {

  }
```

rather than:

```java
  public void f(int a1,
    int a2) {

  }
```